dockercfg-generator/Dockerfile.test

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
+MAINTAINER maintainers@codeship.com
+
+RUN \
+  echo "Hello ECR at $(date)" > hello.txt && \
```

Comment:
Maybe we could change this `hello` message to be less specific (or hello heroku, instead)?